spec/vast_client.spec.js

```diff
@@ -161,7 +158,7 @@ describe('VASTClient', () => {
           expect(res.ads).toHaveLength(3);
         });
 
-        it('resolves only next ad if requested', async () => {
+        fit('resolves only next ad if requested', async () => {
```

Comment:
Perhaps we can create a ticket to add an ESLint rule to prevent fit from being pushed to production?
```suggestion
        it('resolves only next ad if requested', async () => {
```